styles/Vocab/word_list/accept.txt

```diff
@@ -0,0 +1,66 @@
+Meilisearch
+subroute
+substring
+uid
+Qovery
+Koyeb
+Vue
+[Gg]eosearch
+Nginx
+Docxtemplater
+config
+Algolia
+Typesense
+Symfony
+Laravel
+Strapi
+[Sc]chemaless
+reindexed
+chatbox
+Lucene
+Levenshtein
+Swiftype
+tokenizer
+tokenization
+[Aa]sync
+[Ee]nqueued
+[Ff]ilepath
+[Ww]ebshop
+boolean
+Taj 
+Mahal
+[Cc]ertbot
+toolchain
+vm
+Jieba
+Lindera
+Netlify
+unintuitive
+overconsumption
+geopositioning
+hanzi
+[Aa]utobatching
+impactful
+lowercased
+[Mm]ultitenancy
+rulesets
+prepended
+deallocated
+anonymized
+serverless
+autoscaling
+Unsetting
+hostname
+Scaleway
+cron
+mkcert
+upvote
+Docsearch
+Orderable
+performant
+Nutch
+subproject
+Doofinder
+'/\((.*?)\)'
```

Comment:
That's an interesting one 👀

Reply:
This is basically ignoring all text in (). E.g., in `/datatypes.md`, “datatypes” would be considered a typo
Now that I think about it, we might need to modify this to only ignore links in (), not all text